### PR TITLE
Update the `internal` changelog to reflect new proxy version override feature

### PR DESCRIPTION
**Pull request by scbedd**
And add release date to enable releasing a new version of this package.

### CODE REVIEW

File: sdk/internal/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Release History
 
-## 1.7.0 (Unreleased)
+## 1.7.0 (2024-04-26)
```
Comment (jhendrixMSFT) [inline sdk/internal/CHANGELOG.md:3]: Can this wait until Monday for release in case things go sideways?

**Comment by scbedd** [comment]
Waiting to merge before Monday. 👍

**Review by kurtzeborn — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
Needs a rebase on `main` and update the `changelog`.

**Comment by scbedd** [comment]
> Needs a rebase on `main` and update the `changelog`.

I updated the changelog earlier, but I just rebased on `main` per the request as well.

**Review by benbp — APPROVED** [review]

**Pull request merged**